SingularityUI/app/components/deployDetail/DeployDetail.jsx

```diff
@@ -346,6 +342,7 @@ function mapStateToProps(state) {
 
   return {
     notFound: state.api.deploy.statusCode === 404,
+    pathname: ownProps.location.pathname,
```

Comment:
hm, not sure how I feel about this. Even though the web api doesn't capitalize pathname, it also doesn't capitalize `onClick`, etc. Maybe we should use `pathName`? @tpetr